cfme/utils/appliance/__init__.py

```diff
@@ -620,13 +640,23 @@ def product_name(self):
             self.log.exception(
                 'appliance.product_name could not be retrieved from REST, falling back')
             try:
-                # We need to print to a file here because the deprecation warnings make it hard
-                # to get robust output and they do not seem to go to stderr
-                result = self.ssh_client.run_rails_command(
-                    '"File.open(\'/tmp/product_name.txt\', \'w\') '
-                    '{|f| f.write(I18n.t(\'product.name\')) }"')
-                result = self.ssh_client.run_command('cat /tmp/product_name.txt')
-                return result.output
+                # TODO: Review this section. Does not work unconfigured
+                # # We need to print to a file here because the deprecation warnings make it hard
+                # # to get robust output and they do not seem to go to stderr
+                # result = self.ssh_client.run_rails_command(
+                #     '"File.open(\'/tmp/product_name.txt\', \'w\') '
+                #     '{|f| f.write(I18n.t(\'product.name\')) }"')
+                # result = self.ssh_client.run_command('cat /tmp/product_name.txt')
+                # return result.output
+
+                res = self.ssh_client.run_command('cat /var/www/miq/vmdb/VERSION')
+                if res.rc != 0:
+                    raise RuntimeError('Unable to retrieve appliance VMDB version')
+                version_string = res.output.strip()
+                if version_string == 'master':
+                    return 'ManageIQ'
+                else:
+                    return 'CFME'
```

Comment:
Why don't we use `self.version` here?

Reply:
Double rest access I guess, I don't necessarily care about that much right now.